src/components/Nodes/NodeInfo/NodeInfoDot.jsx

```diff
@@ -0,0 +1,220 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import styled, { css, keyframes } from 'styled-components'
+import PropTypes from 'prop-types'
+import moment from 'moment'
+import PieChart from 'react-minimal-pie-chart'
+
+class NodeInfoDot extends Component {
+  static propTypes = {
+    client: PropTypes.object,
+    /** If component is stickied to apply drop shadow on dot */
+    sticky: PropTypes.bool
+  }
+
+  static isNewBlock(prevProps, newProps) {
+    if (prevProps.client.blockNumber !== newProps.client.blockNumber) {
+      return true
+    }
+    return false
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      pulseColor: '',
+      diffTimestamp: moment().unix()
+    }
+  }
+
+  componentDidMount() {
+    // NOTE: This component should update diff every second
+    this.diffInterval = setInterval(() => {
+      this.setState({ diffTimestamp: moment().unix() })
+    }, 1000)
+  }
+
+  componentDidUpdate(prevProps) {
+    this.pulseIfNewBlock(prevProps)
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.diffInterval)
+  }
+
+  pulseIfNewBlock(props) {
+    const { client } = this.props
+    const { network } = client
+    // If new block arrived, add animation to light
+    if (NodeInfoDot.isNewBlock(props, this.props)) {
+      let pulseColor
+
+      if (network === 'main') {
+        pulseColor = 'green'
+      } else {
+        pulseColor = 'blue'
+      }
+
+      this.setState({ pulseColor }, () => {
```

Comment:
Getting this error almost immediately after turning on the node this morning:
![Screen Shot 2019-03-12 at 9 42 17 AM](https://user-images.githubusercontent.com/3621728/54214347-aa5f9d00-44ab-11e9-8f83-099a089069fe.png)